composer.json

```diff
@@ -25,23 +26,43 @@
                 }
             }
         },
-        {
+        "drupal/webform_permissions_by_term": {
```

Comment:
This doesn't seem right. The drupal/webform_permissions_by_term module should not be used at all.

Reply:
Nice catch! Fixed in <https://github.com/itk-dev/os2forms_digital_post/pull/23/commits/2aa506834b93b5735cf070e7782ee4e515172394>.